### DISCUSSION TITLE
Laravel upgrade

### DISCUSSION THREAD

**Discussion by dmitrijsmihailovs** [rectorphp/rector#7132] [Q&A]
Hello,

I have a project that is using Laravel 5.5 and PHP7.1 at the moment. I would like to upgrade it to Laravel 6 and PHP7.4 for time being.
What would be correct way of going about this?
Should I set laravel/framework to 6.*, run composer update and then run rector with Laravel and PHP7.4 rules?

**Comment by TomasVotruba** [comment]
Hi,

in my experince it's the best to go many little steps rather than one big - one version at a time.

You can use similar approach to any other framework. I wrote a little tutorial that can be helpful: https://tomasvotruba.com/blog/2019/02/28/how-to-upgrade-symfony-2-8-to-3-4/
